SSSOM.md

```diff
@@ -491,3 +492,77 @@ Not part of spec, clean up.
 * Can we have more complex logical mappings? A common use case is a taxon-specific anatomy ontology (taxon X). We want to map A to B, but say that B is not equivalent to A, rather it is equivalent to an OWL expression based on A (A and 'in taxon' some X). But there are plenty of arbitrary expressions that could be supported. A while back I looked at some SNOMED to Uberon mappings; it seemed like a lot of the SNOMED terms would better be ('part of' some UBERON:X) rather than directly equivalent to X.
 *  explicitly declare the metadata elements as equivalent to properties?
 * What about mapping to Literals in general? Is this in-spec?
+
+<a name="minimum"></a>
+
+### Minimum Metadata Recommendation
+
+Current mappings are extremely hard to use for data integration, because they are:
+
+- non-transparently imprecise
+- non-transparently incomplete
+- inaccurate
+- unFAIR
+
+In principle, to reach full integration through mappings, you will have to cross-map all ontologies, or semantic
+spaces, which means if you have N "spaces", you have N x N (-1) mappings. To mitigate the explosion
```

Comment:
either `N choose 2` or `N x (N-1) / 2`

Reply:
Its not even `N x (N-1) / 2` -> because you cant actually walk back!